source/ShipInfoDisplay.cpp

```diff
@@ -303,6 +318,27 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 	heatTable.push_back(Format::Number(60. * emptyMass * .1 * attributes.Get("heat dissipation")));
 	// Pad by 10 pixels on the top and bottom.
 	attributesHeight += 30;
+	
+	// Flight check warnings
+	if(ship.IsYours())
```

Comment:
Is this check needed? Surely one would want to know if there are flight warnings for any ship they can see the information of?

Reply:
The check makes ships for sale in the shipyard (or the map's shipyard) not show that particular line.

Reply:
Yes, I'm aware. If a stock ship being sold is not ready to fly, that should be communicated to the player. This is beneficial for plugin-makers, as a content sanity check, or as a gameplay mechanic (e.g. selling discount ships which are missing parts).

Reply:
Good point, will remove the check.